website/blog/2023-11-06-differentiate-prod-and-staging-environments.md

```diff
@@ -0,0 +1,73 @@
+
+But as companies' dbt deployments have grown more complex, it doesn't make sense to assume that a single job is enough anymore. We need to exchange a job-oriented strategy for a more mature and scalable environment-centric view of the world. To support this, a recent change in dbt Cloud enables project administrators to mark one of their environments as the Production environment, just as has long been possible for the Development environment.
+
+Explicitly separating your Production workloads lets dbt Cloud be smarter with the metadata it creates, and is particularly important for two new features: dbt Explorer and the revised CI workflows.
+
+## dbt Explorer can always have the freshest information available
+
+**The old way**: Your dbt docs site was based on a single job's run.
+
+**The new way**: dbt Explorer uses metadata from across every invocation in a defined Production environment to build the richest and most up-to-date understanding of your project.
+
+Because dbt docs could only be updated by a single predetermined job, users who needed their documentation to immediately reflect changes deployed throughout the day (regardless of which job executed them) would find themselves forced to run a dedicated job which did nothing other than run `dbt docs generate` on a regular schedule.
+
+The Discovery API that powers dbt Explorer ingests all metadata generated by any dbt invocation, which means that it can always be up to date with the applied state of your project. However it doesn't make sense for dbt Explorer to show docs based on a PR that hasn't been merged yet.
+
+To avoid this conflation, you need to mark an environment as the Production environment. All runs completed in _that_ environment will contribute to dbt Explorer's, while others will be excluded. (Future versions of Explorer will support environment selection, so that you can preview your documentation changes as well.)
+
+## Run Slimmer CI than ever with environment-level deferral
+
+**The old way**: [Slim CI](/guides/orchestration/set-up-ci/in-15-minutes) deferred to a single job, and would only detect changes as of that job's last build time.
+
+**The new way**: Changes are detected regardless of the job they were deployed in, removing false positives and overbuilding of models in CI.
+
+Just like dbt docs, relying on a single job to define your state for comparison purposes leads to a choice between unnecessarily rebuilding models which were deployed by another job, or creating a dedicated job that runs `dbt compile` on repeat to keep on top of all changes.
+
+By using the environment as the arbiter of state, any time a change is made to your Production deployment it will immediately be taken into consideration by subsequent Slim CI runs.
+
+## The easiest way to break apart your jobs
```

Comment:
This is great and probably the most important part of your post - would the flow work if we were to move this up to just below the intro so as not to bury the lede?

Reply:
I've changed the link in the bottom line section up top to link directly to this section - I think rearranging to explain how without first explaining why probably isn't ideal?